book/src/keymap.md

```diff
@@ -218,6 +218,8 @@ Jumps to various locations.
 | `n`   | Go to next buffer                                | `goto_next_buffer`         |
 | `p`   | Go to previous buffer                            | `goto_previous_buffer`     |
 | `.`   | Go to last modification in current file          | `goto_last_modification`   |
+| `j`   | Go down file/logical (vs visual) line            | `move_line_down`           |
+| `k`   | Go up file/logical (vs visual) line              | `move_line_up`             |
```

Comment:
```suggestion
| `j`   | Move down textual (instead of  visual) line            | `move_line_down`           |
| `k`   | Move up textual (instead of visual) line              | `move_line_up`             |
```

I think we should also call it move down here to make it clear that this is the same as `j`/`k`. And I would call it `textual` line instead of `file/logical` line

Reply:
Yes, that seems better to me too